src/Sarif.Multitool.Library/Sarif.Multitool.Library.csproj

```diff
@@ -23,10 +23,6 @@
     <PackageReference Include="Microsoft.Json.Schema.Validation" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
     <PackageReference Include="System.Threading.Channels" />
-
-    <!-- We have to ship pre-patch versions of NewtonSoft for VisualStudio SDK. 
-         We mitigate risk by limiting nesting depth. -->
```

Comment:
>limiting [](http://example.com/codeflow?start=29&length=8)

Despite this comment that we mitigated risk by limiting nest depth, I see no evidence of the code that we have done that (by explicitly setting a depth level on serializer objects).